src/lib/core/portal/portal.ts

```diff
@@ -76,15 +77,15 @@ export class ComponentPortal<T> extends Portal<ComponentRef<T>> {
    * This is different from where the component *renders*, which is determined by the PortalHost.
    * The origin is necessary when the host is outside of the Angular application context.
    */
-  viewContainerRef: ViewContainerRef;
+  viewContainerRef?: ViewContainerRef;
 
   /** [Optional] Injector used for the instantiation of the component. */
-  injector: Injector;
+  injector?: Injector;
 
   constructor(
       component: ComponentType<T>,
-      viewContainerRef: ViewContainerRef = null,
-      injector: Injector = null) {
+      viewContainerRef?: ViewContainerRef,
+      injector?: Injector) {
```

Comment:
Sorry, I accidentally omitted the word "also". Meant that we could do
`viewContainerRef?: ViewContainerRef | null,` so that either `undefined` or `null` can be passed in cases when you don't care for that argument.